backend/src/main/java/gov/cdc/usds/simplereport/service/TimeOfConsentService.java

```diff
@@ -0,0 +1,28 @@
+package gov.cdc.usds.simplereport.service;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gov.cdc.usds.simplereport.db.model.PatientLink;
+import gov.cdc.usds.simplereport.db.model.TimeOfConsent;
+import gov.cdc.usds.simplereport.db.repository.TimeOfConsentRepository;
+
+@Service("timeOfConsentService")
```

Comment:
The label is not usually useful unless you're injecting by name (e.g. in a SpEL string).